### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-02-22

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#139] [General]
Previous agenda: #133

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
- [x] Review status of ([next release](https://github.com/orgs/space-ros/projects/3/views/8))
- [ ] Open next meeting discussion

**Comment by xfiderek** [comment]
I won't be able to attend but I have 2 topics that I would like to highlight
1. CI fails for [space_nav2](https://github.com/space-ros/docker/actions/runs/7967568376). Unfortunately I won't manage to find time this week to fix it. I can check it next week.
2. I tried to run IKOS inside spaceros container from tutorials and encountered some issues. I created PRs that fix them, see https://github.com/space-ros/docker/issues/138. Note that we need to push changes to `ament_cobra` and `ament_ikos` repos to fix problems.

**Comment by bailaC** [comment]
@Bckempa I would like to attend the meeting if it is ok. Please let me know how I can participate.

**Reply by ivanperez-keera** [reply]
@bailaC can you send me your email address (ivan DOT perezdominguez AT nasa DOT gov). Thanks!

**Comment by ivanperez-keera** [comment]
Decisions made:
- April 20 will be the deadline for merges for the next release. 
- Matt will be the release manager for the next release.

**Comment by Bckempa** [comment]
Next meeting discussion opened at #141
